src/pages/kb/user-guide/alerts/setting-up-an-alert.md

```diff
@@ -14,43 +15,112 @@ title: Setting Up An Alert
 slug: setting-up-an-alert
 order: 1
 ---
-In the Alerts view, you'll see all existing alerts.
+Alerts provide a convenient way for Redash to notify you when your scheduled queries match some arbitrary criteria. They are often used to monitor system statuses, financial goals, and unusual account activity. Through integrations with Zapier and IFTTT they can also trigger business processes like user on-boarding work flows or support tickets.
+
+You can see all of your existing alerts by clicking `Alerts` on the main navigation menu. This will show you all the alerts in your organization by name, user, current state, and created date.
 
 ![](/assets/images/docs/gitbook/alerts.png)
 
-To set up a new alert you'll need to follow these steps:
+# Usage
+
+Create a new Alert by clicking the blue `Create` button in the navigation menu and selecting `Alert`.
+
+![](/assets/images/docs/gitbook/create-alert.png)
+
+Find the query that you would like to monitor using the search bar. Keep in mind that Alerts do not work with queries that use parameters.
+
+![](/assets/images/docs/gitbook/new-alert-query-search.png)
+
+{% callout info %}
+
+If you set an Alert for a query that returns more than one row of data, the Alert will only read the first row of data.
+
+{% endcallout %}
+
+Use the settings panel to configure your alert:
+
+![](/assets/images/docs/gitbook/alerts_settings.png)
+
+* **Value Column** is the column in your query return that you would like to watch.
```

Comment:
"query return"? Dunno if typo or my en_US needs a version bump.

Reply:
Hmm, I see you used it again a few lines down. Query return. Query return. I'll get used to it eventually 😵

Reply:
Maybe it was supposed to be `query result`? :-)

Reply:
I switched it to "query result".